samples/usb/main.c

```diff
@@ -0,0 +1,37 @@
+#include <hal/debug.h>
+#include <hal/video.h>
+#include <windows.h>
+#include <usbh_lib.h>
+
+/*
+ * This demo will initialise the usb stack, detect events on the usb ports
+ * and print the device PID and VID.
+*/
+void device_connection_callback(UDEV_T *udev, int status)
+{
+    debugPrint("Device connected on port %u (PID: %04x VID: %04x)\n", udev->port_num,
+                                                                      udev->descriptor.idProduct,
+                                                                      udev->descriptor.idVendor);
+}
+
+void device_disconnect_callback(UDEV_T *udev, int status)
+{
+     debugPrint("Device disconnected on port %u (PID: %04x VID: %04x)\n", udev->port_num,

```

Comment:
nit: I think I might prefer order 1. VID, 2. PID; but I'm not sure what other common USB tools use. (Idea is to visually group them by manufacturer first.. so that should come first)

Reply:
Makes sense to me. Updated this